README.md

```diff
@@ -18,66 +45,54 @@ This module will help you:
+|**`logLevel`**|One of: `info`, `warn`, `error`, `silent`|Default: `info`. Used to control how much details the plugin outputs.|
 
-// ...
-plugins: [new BundleAnalyzerPlugin()]
-// ...
-```
+<h2 align="center">Size definitions</h2>
 
-`BundleAnalyzerPlugin` constructor can take an optional configuration object that defaults to this:
+webpack-bundle-analyzer reports three values for sizes. `defaultSizes` can be used to control which of these is shown by default. The different reported sizes are:
 
-```js
-new BundleAnalyzerPlugin({
-  // Can be `server`, `static` or `disabled`.
-  // In `server` mode analyzer will start HTTP server to show bundle report.
-  // In `static` mode single HTML file with bundle report will be generated.
-  // In `disabled` mode you can use this plugin to just generate Webpack Stats JSON file by setting `generateStatsFile` to `true`.
-  analyzerMode: 'server',
-  // Host that will be used in `server` mode to start HTTP server.
-  analyzerHost: '127.0.0.1',
-  // Port that will be used in `server` mode to start HTTP server.
-  analyzerPort: 8888,
-  // Path to bundle report file that will be generated in `static` mode.
-  // Relative to bundles output directory.
-  reportFilename: 'report.html',
-  // Module sizes to show in report by default.
-  // Should be one of `stat`, `parsed` or `gzip`.
-  // See "Definitions" section for more information.
-  defaultSizes: 'parsed',
-  // Automatically open report in default browser
-  openAnalyzer: true,
-  // If `true`, Webpack Stats JSON file will be generated in bundles output directory
-  generateStatsFile: false,
-  // Name of Webpack Stats JSON file that will be generated if `generateStatsFile` is `true`.
-  // Relative to bundles output directory.
-  statsFilename: 'stats.json',
-  // Options for `stats.toJson()` method.
-  // For example you can exclude sources of your modules from stats file with `source: false` option.
-  // See more options here: https://github.com/webpack/webpack/blob/webpack-1/lib/Stats.js#L21
-  statsOptions: null,
-  // Log level. Can be 'info', 'warn', 'error' or 'silent'.
-  logLevel: 'info'
-})
-```
+### `stat`
+
+This is the "input" size of your files, before any transformations like
+minification.
+
+It is called "stat size" because it's obtained from Webpack's
+[stats object](https://webpack.js.org/configuration/stats/).
+
+### `parsed`
+
+This is the "output" size of your files. If you're using a Webpack plugin such
+as Uglify, then this value will reflect the minified size of your code.
 
-### As a CLI utility
-You can also analyze already existing bundles if you have a Webpack Stats JSON file.
+### `gzip`
+
+This is the size of running the file(s) through gzip compression.
```

Comment:
Let's change it to something like this: `This is the size of running the parsed bundles/modules through gzip compression.`